tools/cli/src/mentat_cli/repl.rs

```diff
@@ -56,14 +104,59 @@ impl Repl {
                         println!("");
                     }
                     break;
-                }
+                },
+                InputError(err) => {
+                    if let Some(err) = err {
```

Comment:
When will we not have an error in the input error? In that case, can we have a different enum?